README.md

```diff
@@ -1,16 +1,37 @@
 # Peregrine
 
-[![Build Status](https://travis-ci.org/uc-cdis/peregrine.svg?branch=master)](https://travis-ci.org/uc-cdis/peregrine)
-[![Codacy Badge](https://api.codacy.com/project/badge/Grade/f6128183864d4e5da5093eb72a3c9c97)](https://www.codacy.com/app/uc-cdis/peregrine?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=uc-cdis/peregrine&amp;utm_campaign=Badge_Grade)
-[![Codacy Badge](https://api.codacy.com/project/badge/Coverage/f6128183864d4e5da5093eb72a3c9c97)](https://www.codacy.com/app/uc-cdis/peregrine?utm_source=github.com&utm_medium=referral&utm_content=uc-cdis/peregrine&utm_campaign=Badge_Coverage)
+![version](https://img.shields.io/github/release/uc-cdis/peregrine.svg) [![Apache license](http://img.shields.io/badge/license-Apache-blue.svg?style=flat)](LICENSE) [![Coverage Status](https://coveralls.io/repos/github/uc-cdis/peregrine/badge.svg?branch=master)](https://coveralls.io/github/uc-cdis/peregrine?branch=master) [![Build Status](https://travis-ci.org/uc-cdis/peregrine.svg?branch=master)](https://travis-ci.org/uc-cdis/peregrine)
 
 Query interface to get insights into data in Gen3 Commons
 
-## Setup
+## Developer Setup
+
+### Run
+
+```bash
+poetry install
+./run.py
+```
+
+### Test
+
+```bash
+python bin/setup_test_database.py --host postgres
+mkdir -p tests/resources/keys; cd tests/resources/keys; openssl genrsa -out test_private_key.pem 2048; openssl rsa -in test_private_key.pem -pubout -out test_public_key.pem; cd -
+```
+
+If needed, set environment variables to point to a specific Postgres instance.
+
+```bash
+export GDC_PG_HOST=postgres
+export GDC_PG_USER=postgres
+export GDC_PG_PASSWORD=""
+```
+
+Run tests.
 
 ```bash
-# Install requirements.
-pip install -r requirements.txt
+py.test -vv --cov=peregrine --cov-report xml tests
```

Comment:
also `poetry run pytest` here?

Reply:
Excellent point. Updated! https://github.com/uc-cdis/peregrine/pull/174/files#diff-6ac3f79fc25d95cd1e3d51da53a4b21b939437392578a35ae8cd6d5366ca5485R31

Reply:
Excellent point. Updated! Thanks for catching that. https://github.com/uc-cdis/peregrine/pull/174/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R34